src/pool/srv_pool.c

```diff
@@ -4808,6 +4808,7 @@ ds_pool_evict_handler(crt_rpc_t *rpc)
 			D_GOTO(out_free, rc);
 
 		ds_pool_iv_srv_hdl_invalidate(svc->ps_pool);
+		ds_iv_ns_leader_stop(svc->ps_pool->sp_iv_ns);
```

Comment:
What happens if a pool isn't being destroyed, and its pool service leader stops (e.g., due to a network glitch or an ENOMEM in rdb/raft)?

Reply:
oh, if the pool leader only step down, only the iv_master_rank will be set -1, then it will go into the retry loop until the new leader is elected and iv master rank is elected. Anyway step down will not come here, and only destory will execute this function.